examples/timer-log-cdi/src/main/resources/application.properties

```diff
@@ -0,0 +1,38 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+#
+# Quarkus
+#
+quarkus.log.file.enable = false
+
+#
+# Quarkus - Camel
+#
+
+# disable build time route discovery
+quarkus.camel.main.routes-discovery.enabled = false
```

Comment:
After #136 was fixed, this should not be necessary for the example to work. Is it really necessary? (Wondering)

Reply:
It is not necessary just something you can do to completely avoid discovery for routes.

Reply:
Related thought: `quarkus.camel.main.routes-discovery.enabled = false` can actually be expressed as `quarkus.camel.main.routes-discovery.exclude = **/*` so maybe we could remove `quarkus.camel.main.routes-discovery.enabled` ?

Reply:
> It is not necessary just something you can do to completely avoid discovery for routes.

Will it not be confusing for the readers?

Note to myself: We should document that beans managed by Arc are not instantiated from a static initializer - a consequence of #136.

Reply:
`quarkus.camel.main.routes-discovery.enabled` remove the needs to perform the discovery step so it gives a little boost to the build processor and it is much easier to remember as it matches the enable/disable patter used for other features and across platforms

Reply:
> > It is not necessary just something you can do to completely avoid discovery for routes.
> 
> Will it not be confusing for the readers?
> 
Added some clarification to the property